src/elisp/treemacs-visuals.el

```diff
@@ -140,11 +140,18 @@ argument."
       (error
        (treemacs-log "Error on highlight, this shouldn't happen: %s" e)))))
 
+(defun treemacs--create-image (file-path)
+  "Load image from FILE-PATH and resize it to the configured icon size."
+  (if (and treemacs-resize-icons (image-type-available-p 'imagemagick))
```

Comment:
To be really safe you should use `(integerp treemacs-resize-icons)`.

Reply:
Good catch, I'll add that.